.github/workflows/nightly-test.yaml

```
@@ -0,0 +1,61 @@
+name: Nightly Latest/Edge Tests
+
+on:
+  # TODO: remove, this is just testing
+  push: 
+    branches: 
+      - KU-91/nightly-runs
+  schedule:
+    - cron: '0 0 * * *'  # Runs every midnight
+
+jobs:
+  test-integration:
+    name: Integration Test ${{ matrix.os }} ${{ matrix.runner == 'Ubuntu_ARM64_4C_16G_01' && 'arm64' || 'amd64' }}
+    strategy:
+      matrix:
+        os: ["ubuntu:20.04", "ubuntu:22.04"]
+        runner: ["ubuntu-20.04", "Ubuntu_ARM64_4C_16G_01"]
```

ok, so this wasn't immediately clear to me until i spent some time thinking about it

how about this

```suggestion
        os: ["ubuntu:20.04", "ubuntu:22.04"]
        arch: ["amd64", "arm64"]
```


and then when you use apply the runs on, use your github expression:

```yaml
runs-on: ${{ matrix.arch == "arm64" && "Ubuntu_ARM64_4C_16G_01" || "ubuntu-20.04" }}
```
```

---

I was thinking it is looking a bit scary in the name. I like this much better!